worker-macros/src/event.rs

```diff
@@ -50,15 +50,10 @@ pub fn expand_macro(attr: TokenStream, item: TokenStream, http: bool) -> TokenSt
             // rename the original attributed fn
             input_fn.sig.ident = input_fn_ident.clone();
 
-            let error_handling = match respond_with_errors {
-                true => {
-                    quote! {
-                        ::worker::Response::error(e.to_string(), 500).unwrap().into()
-                    }
-                }
-                false => {
-                    quote! { panic!("{}", e) }
-                }
+            let error_handling = if respond_with_errors {
+                quote! { ::worker::Response::error(e.to_string(), 500).unwrap().into() }
+            } else {
+                quote! { ::worker::Response::error("INTERNAL SERVER ERROR", 500).unwrap().into() }
```

Comment:
Should this `console_error`? I think it makes sense to not return the error to the client, but I'm wondering if the it is always desirable to log it somewhere.